OsmSharpMacMSTest/NonAuthTests.cs

```diff
@@ -0,0 +1,230 @@
+using Microsoft.Extensions.Logging;
```

Comment:
This class seems to have nothing to do with overpass, and might be a duplicate of another class?